test/queryplan/queryPlanZoom.html

```diff
@@ -20,7 +20,7 @@
 			var imageBasePath = './icons/';
 			var iconPaths = getIconPaths(imageBasePath);
 
-			let queryPlan = new azdataQueryPlan(container, graph, iconPaths);
+			queryPlan = new azdataQueryPlan(container, graph, iconPaths);
```

Comment:
Why is `queryPlan` being made a global?

Reply:
So that I can access the object through debug console without setting any breakpoints.